examples/ChatQnA/deploy/k8s_helm.md

```diff
@@ -0,0 +1,572 @@
+# Multi-node on-prem deployment with  TGI on Xeon Scalable processors on a K8s cluster using Helm Charts
+
+This deployment section covers multi-node on-prem deployment of the ChatQnA
+example with OPEA comps to deploy using the TGI service. There are several
+slice-n-dice ways to enable RAG with vectordb and LLM models, but here we will
+be covering one option of doing it for convenience : we will be showcasing  how
+to build an e2e chatQnA with Redis VectorDB and neural-chat-7b-v3-3 model,
+deployed on a Kubernetes cluster. For more information on how to setup a Xeon based Kubernetes cluster along with the development pre-requisites,
+please follow the instructions here (*** ### Kubernetes Cluster and Development Environment***). 
+For a quick introduction on Helm Charts, visit the helm section in  (**getting started**)
+
+## Overview
+
+There are several ways to setup a ChatQnA use case. Here in this tutorial, we
+will walk through how to enable the below list of microservices from OPEA
+GenAIComps to deploy a multi-node TGI megaservice solution.
+> **Note:** ChatQnA can also be deployed on a single node using Kubernetes, provided that all pods are configured to run on the same node.
+
+1. Data Prep
+2. Embedding
+3. Retriever
+4. Reranking
+5. LLM with TGI
+
+## Prerequisites
+
+### Install Helm
+First, ensure that Helm (version >= 3.15) is installed on your system. Helm is an essential tool for managing Kubernetes applications. It simplifies the deployment and management of Kubernetes applications using Helm charts. 
+For detailed installation instructions, please refer to the [Helm Installation Guide](https://helm.sh/docs/intro/install/)
+
+### Clone Repository 
+First step is to clone the GenAIInfra which is the containerization and cloud native suite for OPEA, including artifacts to deploy ChatQnA in a cloud native way.
```

Comment:
We already had what was said to be a First in the step above, so maybe "Next step..."